playground/pages/index.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div>
+    <pre> {{ result?.hits }}</pre>
+    <pre>{{ searchForFacetValuesResult }}</pre>
+    <div>
+      <h1>Instantsearch plugin</h1>
+      <ais-instant-search :index-name="indexName" :search-client="algolia">
+        <ais-search-box />
+        <ais-hits />
+      </ais-instant-search>
+    </div>
+    <pre>{{ recommendResult?.results }}</pre>
+
+    <div v-if="algoliaConfig.docSearch">
+      <h3>DocSearch plugin</h3>
+
+      <DocSearch :options="algoliaConfig" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+const indexName = 'test_index'
+
+const { algolia: algoliaConfig } = useRuntimeConfig()
```

Comment:
In order to make it work correctly as described in the docsearch we need to also create docsearch options like following 

```ts
const { algolia: algoliaConfig } = useRuntimeConfig()

const docSearchOptions = { indexName: 'docsearch', apiKey: algoliaConfig.apiKey, appId: algoliaConfig.applicationId }
```